packages/icon/src/IconBase.ts

```diff
@@ -36,6 +36,7 @@ export class IconBase extends SpectrumElement {
         if (changes.has('label')) {
             if (this.label) {
                 this.removeAttribute('aria-hidden');
+                this.setAttribute('aria-label', this.label);
```

Comment:
Great add. However, this whole `updated` method probably should be moved to `Icon.ts` instead. Some `IconBase.ts` extensions will pass the label data into an icon literal, which also manages `aria-hidden`, however these management would be beneficial to the `<sp-icon>` element as it accepts a slotted icon rather than owning icon DOM itself.